Bottom: 647eb27fc68a011bbce9d5f3051424ab36bbfa63
Top:    bd7fc6913845a91ab87801e3e2f767bfe26a4c86
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:36:53 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 27187c1..870afc4 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -1,5 +1,4 @@
 #include <stdio.h>
-#include <stdlib.h>
 #include <stdbool.h>
 
 #include <glib.h>
